frontend/src/views/PollView.vue

```diff
@@ -125,7 +125,7 @@ async function doVote(): Promise<void> {
     // Submit voting request to get signed transaction
     const gasPrice = await uwdao.value.provider.getGasPrice();
     console.log('doVote.gasless: constructing tx', 'gasPrice', gasPrice);
-    const tx = await gv.makeVoteTransaction(gasPrice, request, rsv);
+    const tx = await ugv.makeVoteTransaction(gasPrice, request, rsv);
```

Comment:
This will reveal your vote to the web3 gateway and node operator now right? But since you need to trust the gateway anyway, it doesn't impact security? And it's still using SSL on the transport layer so no MITM attack on wire is possible.

Reply:
Yup you're right about that. It's not ideal...

But couldn't figure out (fairly quickly) how to get it to use the sapphire wrapped provider now that I have a signer attached - as it assumes I want to do signed queries for all view calls and there's no option to 'just do encryption, no signed queries' when submitting requests through ethers without creating a whole new instance of the contract attached to a sapphire provider that's kept isolated from the signer.

Reply:
Maybe you can try with `VoidSigner`. Example here https://github.com/oasisprotocol/demo-authzn/blob/292f84c596c325e5a9ef4f2e63a67317f6e940b1/demo/src/providers/EthereumProvider.tsx#L74

Reply:
Ah sorry, the above is for transfer. Here is better example: https://github.com/oasisprotocol/dapp-wrose/blob/892cd264b0ff6cdd5f72b11dae13bf519dceca90/src/providers/Web3Provider.tsx#L201-L202